coops-ui/tests/unit/FilingHistoryList.spec.ts

```diff
@@ -387,4 +389,22 @@ describe('FilingHistoryList', () => {
       done()
     })
   })
+
+  it('displays the alert when the filing is future effective', done => {
+    const $route = { query: { 'filing_id': '9873' } }
+
+    // init store
+    store.state.entityType = 'BC'
+    store.state.entityIncNo = 'BC0001191'
+    store.state.filings = sampleFilings
+
+    const wrapper = shallowMount(FilingHistoryList, { store, mocks: { $route }, vuetify })
+    const vm = wrapper.vm as any
+
+    expect(vm.$el.querySelectorAll('.filing-history-item')[5].textContent)
+      .toContain('The updated office addresses will be legally effective on 2019-12-12')
```

Comment:
should be 13 I think as per the test data. Please check

Reply:
You're 100%, i missed this after updating. Thank you!